src/system_descriptor_data.jl

```diff
@@ -5,202 +5,510 @@ const SYSTEM_CATALOG = [
         category = PSITestSystems,
         raw_data = joinpath(DATA_DIR, "psy_data", "data_14bus_pu.jl"),
         build_function = build_c_sys14,
+        supported_arguments = [
+            SystemArgument(
+                :add_forecasts,
+                true,
+                Set([true, false]),
+            ),
+        ],
     ),
     SystemDescriptor(;
         name = "c_sys14_dc",
         description = "14-bus system with DC line",
         category = PSITestSystems,
         raw_data = joinpath(DATA_DIR, "psy_data", "data_14bus_pu.jl"),
         build_function = build_c_sys14_dc,
+        supported_arguments = [
+            SystemArgument(
```

Comment:
Can you revert your last commit? I think this is not a good outcome. Sorry for the confusion.